### PR TITLE
Added relaxedPath xml attributes as per jira apache tomcat upgrade file

### PR DESCRIPTION
**Pull request by domgoodwin**
Notes:
https://confluence.atlassian.com/jirasoftware/jira-software-7-13-x-upgrade-notes-957135908.html

#### Pull Request (PR) description
Add relaxedPath attributes into tomcat config to support Jira 7.13+

### CODE REVIEW

File: templates/server.xml.erb [hunk 1]
```diff
@@ -46,6 +46,8 @@
 
     <Service name="Catalina">
         <Connector port="<%= @tomcat_port %>"
+                   relaxedPathChars="[]|" 
+                   relaxedQueryChars="[]|{}^&#x5c;&#x60;&quot;&lt;&gt;"
```
Comment (bastelfreak) [inline templates/server.xml.erb]: there are some really strange characters at the end. Is this on purpose?

Comment (bastelfreak) [reply templates/server.xml.erb]: ah okay, that's even mentioned in their docs.

Comment (domgoodwin) [reply templates/server.xml.erb]: I thought that at first but it's exactly what's in the upgrade docs

Comment (ghoneycutt) [reply templates/server.xml.erb]: a comment that the characters are ```\`"<>``` would be good along with why you would want that

**Comment by vrmerlin** [comment]
I hope this can be added soon -- it's preventing an upgrade to JIRA 8.0+.

**Comment by bastelfreak** [comment]
@vrmerlin the tests are failing on this one, we cannot merge it. But you can always checkout the branch, update it, and provide it as another PR.

**Comment by vrmerlin** [comment]
@bastelfreak  Yes, I understand. I wasn't the original person who made the pull request, so i didn't want to step on Dom's toes if he was working on it.

@domgoodwin are you working on a fix to the PR?

**Comment by domgoodwin** [comment]
Honestly I wasn't sure on the issues raised from the Travis CI build. @vrmerlin feel free to update 👍

**Comment by vrmerlin** [comment]
No idea what i'm doing either, but I'll play around with it.

**Comment by vrmerlin** [comment]
@domgoodwin the failure I"m seeing is this:
        is_expected.to contain_file('/opt/jira/atlassian-jira-6.3.4a-standalone/conf/server.xml').
          with_content(%r{<Connector port=\"9229\"\s+address=\"127\.0\.0\.1\"\s+maxThreads=}m)

I think all you need to do is move your two "relaxedPathChars" entries down further in the list, past the maxThreads, so that this rule still passes. Can you make that change? It might go a lot faster that way.

**Comment by vrmerlin** [comment]
It passed all the tests now!! :-)  Thanks for the second fix, @domgoodwin .

Can this be approved and merged now? I'm giving a demo to another team in a few days, on how easy a JIRA upgrade is...

**Comment by domgoodwin** [comment]
Thanks for the assistance spotting that issue @vrmerlin 🥇

**Comment by vrmerlin** [comment]
@bastelfreak  FYI, I pulled down @domgoodwin 's branch, and used it to do an upgrade to JIRA 8.0, and it worked great!

**Comment by bastelfreak** [comment]
thanks everybody!

**Pull request merged**